### ISSUE TITLE
[Bug] Custom variable of type date does not get localized

### ISSUE THREAD

**Issue by Zeratoxx** [joplin/plugin-templates#68]
## Issue
The date of a custom variable of the type date does not get formatted/localized to the configuration set in Joplin for date format.

## Joplin configuration
`Tools > Options > General > Date Format` is set to: `30.10.2017` (German date format).

## Template
```md
---
set_date: date

---

#  Tagesbericht {{set_date}}
```

###### Generated
```md
#  Tagesbericht 18.07.2022
```

###### Generated
```md
#  Tagesbericht 2022-07-19
```

**Comment by rit-clone** [comment]
Hi @Zeratoxx , @nishantwrp  I would love to work on this issue. Wil you please assign it to me?

**Comment by nishantwrp** [comment]
Sure @rit-clone, feel free to comment if you're facing any issue.

**Comment by rit-clone** [comment]
@nishantwrp I am unable to run the code locally. Can you please provide a way for me to run it so that I can recreate the bug?

**Comment by nishantwrp** [comment]
@rit-clone you can refer this [doc](https://github.com/joplin/plugin-templates/blob/master/GENERATOR_DOC.md).

**Comment by hstandaert** [comment]
I'm also facing this issue, it's just using the default _YYYY-MM-DD_ format. Temporary workaround is to define the format using the `custom_datetime` variable. In my case for _DD/MM/YYY_:

```handlebars
{{#custom_datetime}}DD[/]MM[/]YYYY{{/custom_datetime}}
```

**Comment by Zeratoxx** [comment]
Hi @hstandaert, as far as I know, `custom_datetime` uses the current datetime and it is not possible to use `custom_datetime` with a custom variable, am I right?

**Comment by hstandaert** [comment]
> I'm also facing this issue, it's just using the default _YYYY-MM-DD_ format. Temporary workaround is to define the format using the `custom_datetime` variable. In my case for _DD/MM/YYY_:
> 
> ```
> {{#custom_datetime}}DD[/]MM[/]YYYY{{/custom_datetime}}
> ```
> > Hi @hstandaert, as far as I know, `custom_datetime` uses the current datetime and it is not possible to use 		`custom_datetime` with a custom variable, am I right?

Good point; I was using this in a meeting template so in most cases this workaround covers my needs. But I updated my previous comment 👍🏻

**Comment by nishantwrp** [comment]
Hi @rit-clone, were you able to make any progress?

**Comment by nishantwrp** [comment]
Released a fix in 2.4.0. (581a1da21e7f8adbce1729c3dd5324c991c3abb2)

**Comment by Zeratoxx** [comment]
> Released a fix in v2.4.0. ([581a1da](https://github.com/joplin/plugin-templates/commit/581a1da21e7f8adbce1729c3dd5324c991c3abb2))

Thank you @nishantwrp ❤️
